src/main/java/org/itsallcode/openfasttrace/maven/TraceMojo.java

```diff
@@ -228,14 +257,38 @@ private ImportSettings createImportSettings()
             getLog().info("Tracing doc directory " + docPath.get());
             settings.addInputs(docPath.get());
         }
-        if (artifactTypes != null)
+        final FilterSettings filterSettings = FilterSettings.builder()
+                .artifactTypes(getFilteredArtifactTypes())
+                .tags(getFilteredTags())
+                .withoutTags(isFilterWithoutTags())
```

Comment:
Ah ok, function `withoutTags()` is already part of OpenFastTrace, found it [here](https://github.com/itsallcode/openfasttrace/blob/159a20ebb89a195f41eed181a8f8ecb44a16521e/api/src/main/java/org/itsallcode/openfasttrace/api/FilterSettings.java#L207)